README.md

```diff
@@ -40,26 +40,27 @@ git clone https://github.com/proximax-storage/sirius-storage-testnet-onboarding.
 
 ## How to run as Drive Owner 
 
-
 ```
 cd sirius-testnet-onboarding/drive
 # start storage drive
-./start-drive.sh
+./start-drive.sh <YOUR_PRIVATE_KEY>
```

Comment:
add an alternate instruction on how to run command without specifying private key on the command line.

Reply:
i've updated script with alternative method where user place their private keys in the script